lib/tasks/allrecipe-recipe_category_relations.rake

```diff
@@ -0,0 +1,73 @@
+desc "scrape the relationship between recipes and a category"
```

Comment:
Wouldn't it be a lot faster to associate category with recipes when you were creating recipes?